src/endpoints/auth/auth.service.ts

```diff
@@ -0,0 +1,27 @@
+import { LoginRequest } from '../../models/login-request';
+import { UserService } from "../user/user.service";
+import { JwtService } from '@nestjs/jwt';
+import { HttpException, HttpStatus, Injectable, UnauthorizedException } from '@nestjs/common';
+
+@Injectable()
+export class AuthService {
+    constructor(private userService: UserService, private jwtService: JwtService) { }
+
+    async signIn(userReq: LoginRequest): Promise<any> {
+
+        const user = await this.userService.findOne(userReq.username);
+        const bcrypt = await require('bcryptjs');
```

Comment:
Remove this and use normal import instead.

Try the following code 

```suggestion
        import  bcryptjs from 'bcryptjs';
```